test/unit/lib/transformer.js

```diff
@@ -596,93 +597,6 @@ test('getLogs', async (t) => {
     }
   })
 
-  t.test('where s3 returns an error', async (t) => {
```

Comment:
these tests still appear to be relevant for the functionality extracted into the wrapper. is there a reason for removing them?

Reply:
I was having trouble testing the new SDK.  Let me give it another try now that I separated it.